EIPS/eip-5218.md

```diff
@@ -0,0 +1,235 @@
+---
+eip: 5218
+title: NFT Rights Management Standard
+description: An interface for creating copyright licenses that transfer with an NFT.
+status: Draft
+type: Standards Track
+category: ERC
+author: James Grimmelmann <james.grimmelmann@cornell.edu>, Yan Ji <jyamy42@gmail.com>, Tyler Kell <sk3259@cornell.edu>
```

Comment:
At least one author must have a GitHub handle.

Reply:
Github handles added. Thank you!